includes/Utilities/Performance_Logger.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * WooCommerce Square
+ *
+ * This source file is subject to the GNU General Public License v3.0
+ * that is bundled with this package in the file license.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0 or later
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@woocommerce.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade WooCommerce Square to newer
+ * versions in the future. If you wish to customize WooCommerce Square for your
+ * needs please refer to https://docs.woocommerce.com/document/woocommerce-square/
+ *
+ * @author    WooCommerce
+ * @copyright Copyright: (c) 2019, Automattic, Inc.
+ * @license   http://www.gnu.org/licenses/gpl-3.0.html GNU General Public License v3.0 or later
+ */
+
+namespace WooCommerce\Square\Utilities;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Performance logging utility for timing operations and memory usage.
+ *
+ * @since {next_version}
```

Comment:
Do we needs to add upcoming release version or it will taken care before release?

cc. @dkotter

Reply:
For any newly added `@since` statements, set those to `x.x.x` and they will be updated during the release process

Reply:
Thanks. Updated in https://github.com/woocommerce/woocommerce-square/pull/279/commits/c2bf61bf3dd729bb7fbd380a8f72d9eb3638dcf0